site/en/docs/extensions/whatsnew/index.md

```diff
@@ -19,8 +19,10 @@ Check this page often to learn about changes to Chrome extensions, extensions do
 
 <p class="color-secondary-text type--caption">Posted on <time>October 9, 2023</time></p>
 
-We published a new guide on [how cookies and web storage APIs work](/docs/extensions/mv3/storage-and-cookies/) in Chrome extensions. It includes all you need to know about
-[Privacy Sandbox](/docs/privacy-sandbox/) as an extension developer.
+We published a new guide on [how cookies and web storage APIs work in Chrome extensions](/docs/extensions/mv3/storage-and-cookies/).
+It includes details on how cookie and storage partitioning changes in
+[Privacy Sandbox](/docs/privacy-sandbox/), an ongoing project to deprecate third-party cookies
+through the creation of a series of new APIs, work in extensions.
```

Comment:
```suggestion
through the creation of a series of new web platform APIs, and work in extensions.
```

Reply:
Accepting this but removed the "and" (it's not that work is being done in extensions, it's that we're telling you how something works/behaves).